### DISCUSSION TITLE
Color styling issue in Livewire component when using form and table builder.

### DISCUSSION THREAD

**Discussion by aimonext** [filamentphp/filament#9403] [Help] [answered]
I've used a form and table builder within a custom Livewire component, but the action or text colors aren't being applied. If modifying the 'tailwind.config.js' file can resolve this issue, what configuration changes should be made?

#### tailwind.config.js
```js
import forms from "@tailwindcss/forms";
import typography from "@tailwindcss/typography";

/* @type {import('tailwindcss').Config} */
export default {
    content: [
        "./vendor/laravel/framework/src/Illuminate/Pagination/resources/views/*.blade.php",
        "./storage/framework/views/*.php",
        "./resources/views/**/*.blade.php",
    ],
    darkMode: "class",
    theme: {
        extend: {
            zIndex: {
                '60': '60',
                '70': '70',
                '80': '80',
                '90': '90',
                '100': '100',
            }
        },
    },
    plugins: [forms, typography],
};
```

### Issues
![Screenshot from 2023-11-01 17-31-51](https://github.com/filamentphp/filament/assets/110348549/8f8d5f20-29a2-4411-8167-1945803db579)
![Screenshot from 2023-11-01 17-31-18](https://github.com/filamentphp/filament/assets/110348549/5653fabf-b488-4146-bd51-4663004b1451)
![Screenshot from 2023-11-01 17-31-12](https://github.com/filamentphp/filament/assets/110348549/48743085-21d7-4e01-9bd0-fa15d81aba19)

**Comment by leandrocfe** [comment, marked as answer]
https://filamentphp.com/docs/3.x/forms/installation/#installing-tailwind-css

`presets: [preset]` ?
